test/test_issue801.py

```diff
@@ -13,7 +13,8 @@ def test_issue_801(self):
         g.bind('', example)
         node = BNode()
         g.add((node, example['first%20name'], Literal('John')))
-        print(g.serialize(format="turtle").decode())
+        self.assertEqual(g.serialize(format="turtle").decode().split("\n")[-3] == '[] :first%20name "John" .', True)
```

Comment:
Can this instead be:

```
self.assertEqual(g.serialize(format="turtle").decode().split("\n")[-3],
                '[] :first%20name "John" .')
```

That way if the test fails you get to see what both sides of the equality comparison are. Plus keeping the line length shorter and making the string easier to see.

Reply:
@darrengarvey I have applied the requested changes